### PR TITLE
Refactor tickets resource by using the update method for PUT requests

### LINKED ISSUE #617: Refactor the put method in the Ticket resource
Refactor the put method in the ticket resource to use the new update method from the base resource class, and remove the reqparse code. Refactor associated tests as necessary

### PR DESCRIPTION
**Pull request by Chris-Boe**
## Description
Refactor tickets resource by using the update method for PUT requests, and uses the request json instead of the parser. Also updates tests.

### What issue is this solving?

Closes codeforpdx/dwellingly-app/issues/617

### CODE REVIEW

**Comment by Chris-Boe** [comment]
Putting a Do Not Merge label on this so this can be the final "cleanup" PR of the tickets resource. After the others are merged, I will rebase this and get rid of the leftover parser arguments and imports.

**Review by NickSchimek — APPROVED** [review]
Looks good. Let's get this merged and you can do the final removal of the request parser in the other pr I requested changes in. Thanks!

**Pull request merged**
